app/src/main/java/ch/epfl/skysync/models/reports/Report.kt

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.skysync.models.reports
+
+import ch.epfl.skysync.models.flight.Vehicle
+import java.sql.Time
+
+interface Report {
+  val begin: Time
+  val end: Time
+  val pause: Boolean
+  val pauseDuration: Long
```

Comment:
You didn't specify the unit of the pause duration (I'm assuming seconds but it would be neat to specify it explicitly).
Also, maybe we could have a single field `pause: Long?` that would be null when no pause have been taken ?

Reply:
It is in milliseconds as in other places where we use time as long but I forgot to precise it. I've done it now.